lib/rpc/request.ex

```diff
@@ -7,7 +7,11 @@ defmodule Soroban.RPC.Request do
 
   alias Soroban.RPC.{Client, Error, HTTPError}
 
-  @default_url "https://rpc-futurenet.stellar.org:443/"
+  @base_urls [
+    test: "https://soroban-testnet.stellar.org",
+    future: "https://rpc-futurenet.stellar.org",
+    local: "http://localhost:8000"
```

Comment:
do we really need the local network?

Reply:
@einerzg Currently `stellar_sdk` already supports local so we didn't find any reason to not include it here.